packages/devtools_app/lib/src/screens/inspector/layout_explorer/ui/widget_constraints.dart

```diff
@@ -17,40 +15,41 @@ import 'utils.dart';
 
 class VisualizeWidthAndHeightWithConstraints extends StatelessWidget {
   VisualizeWidthAndHeightWithConstraints({
-    @required this.properties,
-    double arrowHeadSize,
-    @required this.child,
+    required this.properties,
+    double? arrowHeadSize,
+    required this.child,
     this.warnIfUnconstrained = true,
   }) : arrowHeadSize = arrowHeadSize ?? defaultIconSize;
 
-  final Widget child;
-  final LayoutProperties properties;
+  final Widget? child;
```

Comment:
make child non-nullable and use a SizedBox instead where null is passed in currently

Reply:
done